price_service/client/js/package.json

```diff
@@ -41,10 +40,10 @@
     "yargs": "^17.4.1"
   },
   "dependencies": {
-    "@pythnetwork/pyth-sdk-js": "^1.2.0",
+    "@pythnetwork/price-service-sdk": "*",
     "@types/ws": "^8.5.3",
-    "axios": "^0.26.1",
-    "axios-retry": "^3.2.4",
+    "axios": "^1.2.5",
```

Comment:
I faced some axios issues when lock file removed in favor of a global lock file. So needed to update everything to a proper version.